extensions/amp-social-share/amp-social-share.md

```diff
@@ -26,12 +26,19 @@ limitations under the License.
 
 The `amp-social-share` component displays a social share button for social platforms.
 
+{% call callout('Important', type='caution') %}
+Currently, none of the buttons generated by `amp-social-share` (including those for the pre-configured providers) don't have a label/accessible name that is exposed to assistive technologies (such as screen readers). Make sure to include an `aria-label` with a descriptive label, as otherwise these controls will just be announced as unlabelled "button" elements.
```

Comment:
nit: double negative "none of the buttons .. don't have"

Reply:
Fixed